tests/unit/test_db_sync.py

```diff
@@ -310,5 +311,35 @@ def test_record_primary_key_string(self, query_patch):
         stream_schema_message['key_properties'] = ['invalid_col']
         dbsync = db_sync.DbSync(minimal_config, stream_schema_message)
         with self.assertRaisesRegex(PrimaryKeyNotFoundException,
-                                    "Cannot find \['invalid_col'\] primary key\(s\) in record\. Available fields: \['id', 'c_str'\]"):
+                                    r"Cannot find \['invalid_col'\] primary key\(s\) in record\. Available fields: \['id', 'c_str'\]"):
             dbsync.record_primary_key_string({'id': 123, 'c_str': 'xyz'})
+
+    @patch('target_snowflake.db_sync.DbSync.query')
+    def test_merge_copy_failure_log_message(self, query_patch):
+        query_patch.return_value = [{'type': 'CSV'}]
+        minimal_config = {
+            'account': "dummy-value",
+            'dbname': "dummy-value",
+            'user': "dummy-value",
+            'password': "dummy-value",
+            'warehouse': "dummy-value",
+            'default_target_schema': "dummy-value",
+            'file_format': "dummy-value"
+        }
+
+        stream_schema_message = {
+            "stream": "public-table1",
+            "schema": {
+                "properties": {
+                    "id": {"type": ["integer"]},
+                    "c_str": {"type": ["null", "string"]}
+                }
+            },
+            "key_properties": ["id"]
+        }
+
+        # Single primary key string
+        dbsync = db_sync.DbSync(minimal_config, stream_schema_message)
+        query_patch.side_effect = Exception()
+        with pytest.raises(Exception, match="Error while executing*"):
+            dbsync.load_file(s3_key="dummy-key", count=256, size_bytes=256)
```

Comment:
This test class inherits from `unittest`. in the other hand we agreed not to use pytest for future tests.  

I suggest to replace it with something like this:

```python
expected_message = f'Error while executing MERGE query for table ' \
                   f'"{minimal_config["default_target_schema"]}."TABLE1""' \
                   f' in stream "{stream_schema_message["stream"]}"'
with self.assertRaises(Exception) as exp:
    dbsync.load_file(s3_key="dummy-key", count=256, size_bytes=256)
self.assertEqual(expected_message, str(exp.exception))
```

Reply:
As a person who is reading this code I don't know what `TABLE1` is!
define a variable and make expected value from uppercase of the second part of `stream` !

Reply:
fixed